Aufnahmeantrag_aktiv.tex

```diff
@@ -62,11 +64,18 @@ \section*{Aufnahmeantrag\\ zur aktiven Mitgliedschaft im \netzEV}
                & \tf{email}{10cm}\\
 E-Mail-Adresse & \dotfill%
 \end{tabularx}
+\end{spacing}
 \end{center}
 
 Die Mitgliedsbeiträge richten sich nach der jeweils gültigen Beschlussfassung der Mitgliederversammlung. Mit meiner Unterschrift erkenne ich die Satzung des \netzEV an und verpflichte mich, die satzungsgemäßen Ziele des \netzEV nach Kräften zu unterstützen.
 
-\vspace{2cm}
+\textbf{Wir weisen gemäß § 33 Bundesdatenschutzgesetz darauf hin, dass zum Zweck der Mitgliederverwaltung und -betreuung folgende Daten der Mitglieder in automatisierten Dateien gespeichert, verarbeitet und genutzt werden: Name, Anschrift, Geburtsdatum und -Ort, E-Mail-Adresse}
+
+\subsection*{Einverständnis zur Datenerhebung und -verarbeitung}
+
+Ich bin mit der Erhebung, Verarbeitung und Nutzung folgender personenbezogener Daten durch den Verein zur Mitgliederverwaltung im Wege der elektronischen Datenverarbeitung einverstanden: Name, Anschrift, Geburtsdatum und -Ort, E-Mail-Adresse. Mir ist bekannt, dass dem Aufnahmeantrag ohne dieses Einverständnis nicht stattgegeben werden kann.
```

Comment:
Können wir hier auch auf "die oben genannten Daten" verweisen, um die Dopplung zu verhindern?

Reply:
"Ich bin mit der Erhebung, Verarbeitung und Nutzung der oben genannten Daten durch den Verein zur Mitgliederverwaltung im Wege der elektronischen Datenverarbeitung einverstanden. Mir ist bekannt, dass dem Aufnahmeantrag ohne dieses Einverständnis nicht stattgegeben werden kann."

Reply:
Stimme tux zu. Können wir das so formulieren, dass der nachfolgende Unterschrift sich nicht nur auf den Einverständnis von Datenverarbeitung bezieht, sondern auf auf den eigentlichen Antrag?

Danke übrigens, dass ihr das jetzt in Angriff genommen habt :)

Reply:
Ich habe gerade oben  einen Kasten erwähnt.... vielleicht machen wir einen Kasten um beides? 
Mir war das auch schon aufgefallen... aber macht der Antrag ansonsten überhaupt Sinn?

Reply:
Ja ein Kasten um beide Sachen herum und dann ein Satz wie : "Mit dem Stellen des Antrages stimme ich automatisch der Datenverarbeitung zu" oder so ähnlich